reconcile/dyn_traffic_director.py

```diff
@@ -7,6 +7,13 @@
 from reconcile.utils.helpers import toggle_logger
 from reconcile.utils.secret_reader import SecretReader
 
+# Dirty hack for Python 3.10 to overcome
```

Comment:
we will want 3.10 eventually. some people are already working with it (myself included). if we want to keep this integration around, and want python 3.10, this is likely the only way forward.

Reply:
The latest version of `dyn` claims to have py3.10 compatibility:

https://github.com/dyninc/dyn-python/releases/tag/1.8.4

Should we try upgrading the package instead?

Reply:
that would be better, indeed! it's not published though: https://pypi.org/project/dyn/

Reply:
https://github.com/dyninc/dyn-python/issues/145 :crossed_fingers: 

Worst case: should we fork? I see JF also has a fix in there that didnt get merged since a while.

Reply:
Then how about fetching straight from github?

**setup.py:**
```python
#"dyn~=1.8.1",
"dyn @ git+https://github.com/dyninc/dyn-python@1.8.4",
```